Cargo.toml

```diff
@@ -14,6 +14,7 @@ skiplist = { path = "skiplist" }
 memmap = "0.7"
 farmhash = "1.1"
 prost = "0.6"
+bitvec = "0.19"
```

Comment:
I suggest not to use `bitvec` as the code is quite simple and `bitvec` can bring another 4 indirect deps.

Reply:
I've implemented a simple bitvec.